packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-resources/translations/Messages_fr_FR.json

```
@@ -3,7 +3,7 @@
   "pci_notebook_add_resources_usecase_item_standard": "AI Notebooks standard  (ressources CPU)",
   "pci_notebook_add_resources_usecase_item_standard_description": "Moins coûteux et proposant des performances standards. Adapté à l’apprentissage, aux tests ou aux workloads nécessitant du CPU uniquement.",
   "pci_notebook_add_resources_usecase_item_advanced": "AI Notebooks avancé (ressources GPU)",
-  "pci_notebook_add_resources_usecase_item_advanced_description": "Alimenté par plus de quatre GPU et des douzaines de CPU. Propose de meilleures performances pour l’apprentissage et les workloads intensifs.",
+  "pci_notebook_add_resources_usecase_item_advanced_description": "Alimenté jusqu'à quatre GPU et des douzaines de CPU. Propose de meilleures performances pour l’apprentissage deep learning et les workloads intensifs.",
```

```suggestion
  "pci_notebook_add_resources_usecase_item_advanced_description": "Alimenter jusqu'à quatre GPU et des douzaines de CPU. Propose de meilleures performances pour l’apprentissage deep learning et les workloads intensifs.",
```
no ? Seems strange past + "jusqu'à"

---

For me "Alimenté" seem's good

---

I definitely don't get the meaning of this sentence. The fact that it is "Alimenté" used as an adjective seems wrong here

---

I got new sentence.
Updated.